packages/server/test/support/fixtures/projects/e2e/cypress/integration/visit_spec.js

```diff
@@ -158,4 +158,10 @@ describe('visits', () => {
       })
     })
   })
+
+  context('issue #15738: Support application/xhtml+xml content-type in cy.visit() command', () => {
+    it('passes', () => {
+      cy.visit('http://localhost:3434/application-xhtml-xml-content-type')
+    })
+  })
```

Comment:
Instead of adding an e2e test (which requires updating snapshots, which is why you see some of the status checks on your PR failing), you can just update the test that already exists here:

https://github.com/cypress-io/cypress/blob/7e718504c9d70f47f8225499cbec2578d72917ee/packages/server/test/integration/server_spec.js#L471-L492

You can run it by using `yarn test-integration server_spec` inside of `packages/server`.

We try to avoid unnecessarily adding new e2e tests for things that don't need to be run inside of an actual Cypress instance.

Reply:
Thanks for the feedback, will migrate the tests to your suggested format